tutorials/c-tiny/fixeddecimal/Cargo.lock

```diff
@@ -5,7 +5,7 @@ version = 3
 [[package]]
 name = "diplomat"
 version = "0.8.0"
-source = "git+https://github.com/rust-diplomat/diplomat?rev=8744ac97162341f347b63131969ad736e1047f6d#8744ac97162341f347b63131969ad736e1047f6d"
+source = "git+https://github.com/rust-diplomat/diplomat?rev=b49d63ec2a50a456b4b6e9d072d3650dca153ff8#b49d63ec2a50a456b4b6e9d072d3650dca153ff8"
```

Comment:
Nit: Try not to include Cargo.lock changes in PRs with other functional changes. This Cargo.lock file might be stale, but it should be fixed in a standalone PR.